sormas-backend/src/main/java/de/symeda/sormas/backend/location/LocationFacadeEjb.java

```diff
@@ -172,13 +172,26 @@ public boolean areDifferentLocation(LocationDto firstAddress, LocationDto second
 			return true;
 		}
 
+		if (Boolean.TRUE.equals(!DataHelper.equal(firstAddress.getFacilityType(), secondAddress.getFacilityType()))) {
```

Comment:
`DataHelper.equal` is not nullable so there is no need to compare to `Boolean.TRUE`

Reply:
changed